packages/transport/src/modules/accounts/AccountController.ts

```diff
@@ -63,6 +64,7 @@ export class AccountController {
     public relationshipTemplates: RelationshipTemplateController;
     private synchronization: SyncController;
     public tokens: TokenController;
+    public publicRelationshipTemplateReferences: PublicRelationshipTemplateReferencesController;
```

Comment:
This list is mainly in alphabetical order, please insert correctly after `messages` and before `relationships`.